packages/call-to-action/index.js

```diff
@@ -37,15 +37,15 @@ function CallToAction({
         )}
         <div className={s.contentAndLinks}>
           {content && (
-            <p
+            <div
```

Comment:
Fixes issue on re-hydration that borked styles. Big thanks to Bryce & Jeff for guiding me to the fix for this one 🙇‍♂️